kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -57,11 +60,22 @@
   @Parameter(names = "--help", help = true)
   private boolean help = false;
 
+  @Parameter(names = "--strict", description = "Run in strict configuration validation mode")
+  private boolean strict = false;
```

Comment:
I am wondering when would it make sense to not use the string mode?

Reply:
Having strict and normal modes was suggested by @jyates in #30. Jesse maybe you can chime in here?

Reply:
Defaulting to strict would be a behavior change from the existing model. I don't want to surprise folks, especially since we haven't settled on any sort of release/versioning scheme. It also "strictly" (heh) correct to have different bootstrap servers between burrow and freshness tracker; it would be odd, but certainly not wrong. 

Allowing strict mode that fails hard and a WARN level error message for lenient mode is the compromise i propose above